README.md

```diff
@@ -129,11 +129,11 @@ With the next command you will create a new table to handle your refresh tokens
 
 **Symfony 3 Version:**   
 ```bash
-php app/console doctrine:schema:update --force
+php bin/console doctrine:schema:update --force
```

Comment:
I don't see now any difference with the commands below described for Symfony 4 version.